framework/base/Module.php

```diff
@@ -356,9 +356,7 @@ protected function defaultVersion()
      * Defines path aliases.
      * This method calls [[Yii::setAlias()]] to register the path aliases.
      * This method is provided so that you can define path aliases when configuring a module.
-     * @property array list of path aliases to be defined. The array keys are alias names
-     * (must start with `@`) and the array values are the corresponding paths or aliases.
-     * See [[setAliases()]] for an example.
```

Comment:
No, see my comment in the commit that reverts it. This annotation is to generate the `@property ` annotation in class PHPdoc for this setter.